### DISCUSSION TITLE
Generic mapping files quota limits--

### DISCUSSION THREAD

**Discussion by gania241** [SalesforceCommerceCloud/sfcc-ci#432] [Team Posts]
Hi Folks,
I have a question for generic mapping files quota limits - what is the maximum limit of csv's we can import via *ImportKeyValueMapping* pipelet?

**Comment by mkimpelsf** [comment]
From https://documentation.b2c.commercecloud.salesforce.com/DOC1/index.jsp?topic=%2Fcom.demandware.dochelp%2FImportExport%2FGenericMapping.html&resultof=%22mapping%22%20%22map%22%20:

Mapping files are limited to 20MB, which is approximately equivalent to 1M records of 20 bytes and a maximum of 20 mappings.

**Comment by gania241** [comment]
Perfect.
